src/pal/inc/pal.h

```diff
@@ -5613,6 +5613,18 @@ PALIMPORT
 DWORD
 PALAPI
 GetCurrentProcessorNumber();
+
+/*++
+Function:
+PAL_HasGetCPU
+
+Checks if sched_getcpu is available in the current environment
```

Comment:
This comment should rather say "Checks if GetCurrentProcessorNumber is implemented in the current environment"